googletest/src/gtest.cc

```diff
@@ -868,6 +872,10 @@ TimeInMillis GetTimeInMillis() {
   struct timeval now;
   gettimeofday(&now, NULL);
   return static_cast<TimeInMillis>(now.tv_sec) * 1000 + now.tv_usec / 1000;
+#elif GTEST_HAS_CHRONO_
```

Comment:
I don't believe that the current #if / #elif combination does what you want with regards to selecting chrono versus other options available here.